dali/test/python/test_operator_random_object_bbox.py

```diff
@@ -252,7 +270,7 @@ def _test_random_object_bbox_with_class(max_batch_size, ndim, dtype, format=None
                                                classes=classes, class_weights=weights, background=background,
                                                threshold=threshold, k_largest=k_largest,
                                                seed=1234)
-        outs1 = op(inp)
+        outs1 = op(inp, cache_objects=cache)
```

Comment:
Whoever would see the mix of ops and fn API would be definitely confused.

Reply:
I think that now it's about the only (?) justifiable use of ops - preconfiguring operators.

Reply:
I know, still it is not that obvious for the anyone encountering this.

Reply:
Maybe use `partial` if you want to pre-configure, to avoid confusion